test/topology_custom/test_tablets_cql.py

```diff
@@ -66,3 +67,65 @@ async def alter_tablets_ks_without_waiting_to_complete():
 
     with pytest.raises(InvalidRequest, match="Can't ALTER keyspace ks, keyspace doesn't exist") as e:
         await task
+
+@pytest.mark.asyncio
+@skip_mode('release', 'error injections are not supported in release mode')
+async def test_alter_tablets_keyspace_concurrent_modification(manager: ManagerClient) -> None:
+    config = {
+        'enable_tablets': 'true'
+    }
+
+    logger.info("starting a node (the leader)")
+    servers = [await manager.server_add(config=config)]
+
+    logger.info("starting a second node (the follower)")
+    servers += [await manager.server_add(config=config)]
```

Comment:
why starting servers one by one and not take 2 node cluster from the beginning instead?

I don't see how you can guarantee here who is raft leader (or more precisely topology coordinator, is this always the same? not sure), the fact that at the beginning it's this node doesn't mean anything few lines of code later

I don't know what is raft idiomatic way to do this but for instance I see "get_topology_coordinator" helper, maybe that's the way.

Reply:
OK, leaving as is, as I want the first node to be the leader/coordinator (this node has to handle the concurrent modification, not the other node) and keeping the code simple